drivers/hmis/app/graphql/types/hmis_schema/ce_opportunity.rb

```diff
@@ -53,6 +59,21 @@ def project_type
       load_ar_association(object, :project).project_type
     end
 
+    def organization_name
+      project = load_ar_association(object, :project)
+      load_ar_association(project, :organization).name
+    end
+
+    def unit
+      owner = load_ar_association(object, :owner)
+      owner if owner.is_a?(Hmis::Unit)
+    end
+
+    def date_available
+      # TODO(#7537) - implement "available after date". Always returns date the referral was created, for now
```

Comment:
My understanding, from the spec's reference to “recency”, is that this is meant to be an amalgamation of the opportunity's creation date with the (future field) date available field. So I've implemented it as `created_at` for now with todos for the future ticket.